tests/test_model_serializer.py

```diff
@@ -490,8 +490,28 @@ class Meta:
 
         expected = dedent("""
             TestSerializer():
-                json_field = JSONField(encoder=None, style={'base_template': 'textarea.html'})
```

Comment:
I think this should be run only for django < 3.1, so this test should be decorate with 
```python
@pytest.mark.skipif(hasattr(models, 'JSONField'), reason='has models.JSONField')
```
and do not handle `decoder` param.
What do you think?

Reply:
It's still valid (though throws a deprecation warning) to import it form django.contrib.postgres in 3.1, so why wouldn't we keep testing for it?

Reply:
Because with django >= 3.1 this will tested twice, and you have to adapt old JSONField to behave like the new (which is not rigth IMHO).